### ISSUE TITLE
训练LOSS下降太快

### ISSUE THREAD

**Issue by xurning** [yuanzhoulvpi2017/zero_nlp#19]
{'loss': 6.4776, 'learning_rate': 1.9914000000000003e-05, 'epoch': 4.17}
{'loss': 2.0588, 'learning_rate': 1.9814000000000004e-05, 'epoch': 8.33}
{'loss': 0.6284, 'learning_rate': 1.9714e-05, 'epoch': 12.5}
{'loss': 0.1956, 'learning_rate': 1.9614000000000002e-05, 'epoch': 16.67}
这个是不是下降太快了？

**Comment by zhaodice** [comment]
样本太少，过拟合了

**Comment by xurning** [comment]
> 样本太少，过拟合了

那要怎么判断是否过拟合呢。。是不是要拆分数据集？
